doc/dev/developer_guide.md

```diff
@@ -48,6 +48,36 @@ $ make lint
 **NOTE** Note that for it is required to install `golangci-lint` locally. For more info see its [doc](https://github.com/golangci/golangci-lint#install)
 
 
+## How the operator-sdk images are built
+
+Each time a Pull request is merged, a new image is built and pushed to the `quay.io/operator-framework` repository. This is done by the Travis deploy stage, which is configured in our [.travis.yml][travis].
+
+See that the images are built with different architectures from master as it is also done in the release process. See in the release Tab of the operator-sdk repository, that each version has different architectures as well for each supported operator type such as Helm and Ansible for example. 
```

Comment:
```suggestion
Each build creates assets of each operator type for multiple architectures. See [operator-sdk releases](https://github.com/operator-framework/operator-sdk/releases).
```

Reply:
At least ^ is what I thought you meant. But when I actually look at the releases tab, I don't see separate assets for Ansible/Helm.